src/dist/js/footernav.js

```diff
@@ -1,28 +1,29 @@
-document.getElementById("lnkNext").addEventListener("click", function () {
+const url = new URL(window.location);
+const currentPage = url.searchParams.get("_getpagesoffset");
+
+const nextLink = document.getElementById("lnkNext");
+nextLink.addEventListener("click", function () {
     goNext();
 }, false);
 
-document.getElementById("lnkNext").addEventListener("click", function () {
```

Comment:
This was the main bug. The id "lnkNext" was incorrectly used for what should have been "lnkPrevious".